### PR TITLE
fix: Use `__conda_activate reactivate` instead of `__conda_reactivate`

### PR DESCRIPTION
**Pull request by hmaarrfk**
The work in https://github.com/mamba-org/mamba/pull/3643 was deleted

this is really hurting users at conda-forge (myself included) https://github.com/conda-forge/miniforge/issues/700#issuecomment-2554272743

### CODE REVIEW

File: mamba/mamba/shell_templates/mamba.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ mamba() {
             ;;
         install|update|upgrade|remove|uninstall)
             __mamba_exe "$@" || \return
-            __conda_reactivate
+            __conda_activate reactivate
```
Comment (jjerphan) [inline mamba/mamba/shell_templates/mamba.sh:20]: Are we sure that `__conda_activate` is always defined?

Comment (JohanMabille) [reply mamba/mamba/shell_templates/mamba.sh:20]: Yes, conda_reactivate was removed because it was duplicating `__conda_activate reactivate`.

**Review by JohanMabille — APPROVED** [review]

**Review by jjerphan — APPROVED** [review]
Thank you, @hmaarrfk.

Let us release 1.5.12, then.

**Comment by wolfv** [comment]
Is `__conda_activate` defined for `micromamba`

**Comment by hmaarrfk** [comment]
i'm not sure.

(this is the one line bug that keeps on giving....)

**Comment by wolfv** [comment]
For context, some users reported infinite hanging with micromamba.https://conda.discourse.group/t/staged-recipes-linter-hangs/930

**Comment by wolfv** [comment]
Should we just delete the micromamba release?

**Comment by hmaarrfk** [comment]
There was a large period of time (nov - Jan 3) where I had lost confidence in conda/mamba because various updates from both kept causing different lockups.

On Friday, I pushed https://github.com/conda-forge/conda-feedstock/pull/251 which fixed a bug in a conda warning, so I'm wondering if that might help current users?

I don't know anything about micromamba (never tried it)

**Comment by jjerphan** [comment]
> For context, some users reported infinite hanging with micromamba.https://conda.discourse.group/t/staged-recipes-linter-hangs/930

The error was due to an out-of-date version of staged-recipe apparently.

**Comment by hmaarrfk** [comment]
I'm glad we all get to live to fight an other day!

**Pull request merged**